build-tools/packages/build-tools/src/common/utils.ts

```diff
@@ -127,7 +127,13 @@ function printExecError(
 				? `${errorPrefix}: ${ret.stdout}\n${ret.stderr}`
 				: `${errorPrefix}: ${ret.stderr}`,
 		);
-	} else if (warning && ret.stderr) {
+	} else if (
+		warning &&
+		ret.stderr &&
+		// tsc-multi writes to stderr even when there are no errors, so this condition excludes that case as a workaround.
```

Comment:
Thanks for the docs! But this is a bummer. Can we file an issue with them (if you haven't already)?

Reply:
https://github.com/tommy351/tsc-multi/issues/22

Reply:
Might be worth adding that link to the comments too.